src/pages/dashboard/daily-customers/NewCustomers.tsx

```diff
@@ -45,7 +32,7 @@ export const DailyCustomerChart = ({ data, height, filter }: Props) => {
   };
 
   const dataConfig = {
```

Comment:
@phanvohieunghia, I think we should rename **dataConfig** to  **chartConfiguration**